user/tests/wiring/ble/heartrate/application.cpp

```diff
@@ -20,21 +20,16 @@
 /* Includes ------------------------------------------------------------------*/
 #include "application.h"
 
-BLEAttribute heartrate(READ | NOTIFY, "heartrate");
-BLEConnection* myConn;
+BLEAttribute heartrate("heartrate", READ | NOTIFY);
+int newHr = 1234;
 
 void setup() {
-    BLE.on();
-
     BLE.advertise();
-
-    BLEDevice local;
-    myConn = BLE.connect();
 }
 
 void loop() {
-    uint32_t newHr = 1234;
-    if (BLE.connected(myConn)) {
-        heartrate.setValue((const uint8_t*)&newHr, sizeof(uint32_t));
+    if (BLE.connected()) {
+        heartrate.setValue(newHr);

```

Comment:
We might need to include the size as a second argument. If possible, I'd love to allow users to omit it.

Reply:
The second argument is required if user wants to pass in a pointer to an array as the first argument.